modules/ossm-cert-manager-install-istio-resource.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="install-istio-resource_{context}"]
+= Install your Istio resource
+
+//TP1 content influx. Title, etc may change.
+//Content is very similar to 2.x content
+//all kinds of formatting things to fix. want to see if a build will generate to have a look, and see how it fits structurally with the IA.
+
+
+After you have installed `istio-csr` by following the procedure for either `InPlace` or `RevisionBased`, you can install the `Istio` resource.
+
+This disables Istio's built in CA server and points istiod to the `istio-csr` CA server. The `istio-csr` CA server issues certificates for both istiod and user workloads. Additionally, the istiod TLS cert is mounted into the pod at a known location where it will be read. is mounted in a known location where it will be read by istiod. Mounting the certificates to a known location is only necessary for {SMProduct}.
```

Comment:
Not clear what "This" is until you scroll down to the example. Maybe can explain here that "We need to disable Istio's built in CA server and instead tell istiod to use the `istio-csr` CA server."

Reply:
> Additionally, the istiod TLS cert is mounted into the pod at a known location where it will be read. is mounted in a known location where it will be read by istiod. Mounting the certificates to a known location is only necessary for {SMProduct}.

I wouldn't bother mentioning this part. It's only necessary because of a bug in TP1 but will be fixed with TP2.